update plan files for ob 0.4.0

btraven00 opened the pull request.

btraven00 committed:
update plan files for ob 0.4.0

markrobinsonuzh commented:
Maybe also worth noting that the repos pointed to in these YAMLs .. or at least https://github.com/imallona/clustbench_data .. is using `config.cfg` still. So, worth updating those too.

btraven00 committed:
fix path for metric collector

kudos to atreya for figuring this out

btraven00 committed:
update data generator commit

btraven00 committed:
no need for wget

btraven00 committed:
update modules to 0.4 metadata

Commit:
Remove {input} prefix from metric collector

markrobinsonuzh committed:
update commit hash to newer report repo

markrobinsonuzh committed:
change it to branch name

btraven00 committed:
should be a string

otherwise, this is interpreted as a number. 040 is 32 in octal, which is
what git was trying to clone.

markrobinsonuzh commented:
Just keep in mind that I'm also working on this PR: 
https://github.com/omnibenchmark/clustering_example/pull/47
